frontend/src/components/tables/Columns/forms/FormResponseColumns.tsx

```
@@ -47,17 +69,98 @@ export const useFormResponseColumns = () =>
+        accessorKey: "submittedAt",
+        header: "נענתה בתאריך",
+        cell: ({ row }) => formatSubmittedAt(row.original.submittedAt),
+      },
+
+      {
+        id: "viewed",
+        header: "נצפה",
         cell: ({ row }) => {
-          return resolveUserName(row.original.userId) || "-";
+          const { toggleIsCheckedResponse } = useFormResponsesApi();
+          const [isViewed, setIsViewed] = useState(Boolean(row.original.isChecked));
```

Biome (2.3.13)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)